### PR TITLE
Feature/theme

### PR DESCRIPTION
**Pull request by evanbutera**

### CODE REVIEW

File: src/Main.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ interface Options {
     webfonts?:string;
     scripts?:any;
     customStylesheet?:string;
+    hightLight?:number;
```
Comment (BrianDGLS) [inline src/Main.ts]: "hightlight" should be spelled "highlight".

File: README.md [hunk 2]
```diff
@@ -175,6 +175,16 @@ A template stylesheet is provided [here](https://github.com/BrianDGLS/node-holog
...
 ```
 
+__hightlight__ `optional`
+
+Add [hightlight.js](https://highlightjs.org/) to the style guide.
+This option is enabled by default.
+
+
+```javascript
+hightlightjs: 1
```
Comment (BrianDGLS) [inline README.md]: Could we switch this to a `true` or `false` value, mainly for clarity also because it's written in TypeScript.

**Comment by BrianDGLS** [comment]
Looks good nice work. 

The only issue for me was that the Ide links don't work. I'm using chrome on OSX.

I really like the idea of the Ide links but I'm unsure if this is something that would work as a default, especially as I can see people complaining that their editor is not represented.

Also some minor comments on the code.

**Comment by evanbutera** [comment]
jetbrains ides support idename:// protocol from the last version (phpstorm 10) for previous versions a plugin is available

textmate has native support for txmt:// protocol

Sublime requires a plugin

next commt will include options and an update the readme.md

**Comment by evanbutera** [comment]
@BrianDGLS give it check, should be ok

**Comment by BrianDGLS** [comment]
Looks good. Merging.

**Pull request merged**